src/admin/content-block/components/ContentBlockPreview/ContentBlockPreview.const.ts

```diff
@@ -48,4 +48,17 @@ export const REPEATABLE_CONTENT_BLOCKS = [
 	ContentBlockType.ProjectsSpotlight,
 ];
 
-export const IGNORE_BLOCK_LEVEL_PROPS = ['position', 'elements', 'blockType', 'padding'];
+export const NAVIGATEABLE_CONTENT_BLOCKS = [
```

Comment:
welke blokken moet hier precies in?
Want de page overview block en de image grid hebben ook links/buttons

Reply:
De blocks die een `navigate` prop op elk item in hun elements array zetten, ik heb de ProjectsSpotlight nu ook aangepast in de comp. lib. om zo te werken.
Dan moeten we minder exceptions schrijven in de client.

Reply:
Als je een betere naam weet, suggesties zijn altijd welkom. Deze is idd mss iets te algemeen 😄

Reply:
@andrych Misschien `NAVIGABLE` ipv `NAVIGATEABLE`. 👀

Reply:
Fixed